detectors/node/opentelemetry-resource-detector-alibaba-cloud/README.md

```diff
@@ -26,21 +26,23 @@ const resource = await detectResources({
 ```
 
-## Available detectors
+## Available detectors & Semantic Conventions
+
+This package uses `@opentelemetry/semantic-conventions` version `1.22+`, which implements Semantic Convention [Version 1.7.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md)
 
 ### Alibaba Cloud Ecs Detector
 
 Populates `cloud` and `host` for processes running on [Alibaba Cloud ECS](https://www.alibabacloud.com/product/ecs). More info about Alibaba Instance Identities can be found [here](https://www.alibabacloud.com/help/en/ecs/user-guide/use-instance-identities).
 
-| Resource Attribute      |  Description                                                    |
-|-------------------------|-----------------------------------------------------------------|
-| cloud.account.id        | Value of `owner-account-id` on Alibaba Cloud                    |
-| cloud.availability_zone | Value of `zone-id` on Alibaba Cloud                             |
-| cloud.platform          | In this context, it's always `alibaba_cloud_ecs`                |
-| cloud.provider          | In this context, it's always `alibaba_cloud`                    |
-| cloud.region            | Value of `region-id` on Alibaba Cloud                           |
-| host.id                 | Value of `instance-id` on Alibaba Cloud                         |
-| host.name               | The hostname for the app, retrieve from the `hostname` endpoint |
-| host.type               | Value of `instance-type` on Alibaba Cloud                       |
+| Resource Attribute      |  Description                                                    | Notes                                      |
+|-------------------------|-----------------------------------------------------------------| ------------------------------------------ |
+| cloud.account.id        | Value of `owner-account-id` on Alibaba Cloud                    | Key: `SEMRESATTRS_CLOUD_ACCOUNT_ID`        |
+| cloud.availability_zone | Value of `zone-id` on Alibaba Cloud                             | Key: `SEMRESATTRS_CLOUD_AVAILABILITY_ZONE` |
+| cloud.platform          | In this context, it's always `alibaba_cloud_ecs`                | Key: `SEMRESATTRS_CLOUD_PLATFORM`          |
+| cloud.provider          | In this context, it's always `alibaba_cloud`                    | Key: `SEMRESATTRS_CLOUD_PROVIDER`          |
+| cloud.region            | Value of `region-id` on Alibaba Cloud                           | Key: `SEMRESATTRS_CLOUD_REGION`            |
+| host.id                 | Value of `instance-id` on Alibaba Cloud                         | Key: `SEMRESATTRS_HOST_ID`                 |
+| host.name               | The hostname for the app, retrieve from the `hostname` endpoint | Key: `SEMRESATTRS_HOST_TYPE`               |
+| host.type               | Value of `instance-type` on Alibaba Cloud                       | Key: `SEMRESATTRS_HOST_NAME`               |
```

Comment:
Based on comments over on #1778 we may decide not to move forward with this Notes section as it is extra to maintain.